core/kazoo_json/src/kz_json.erl

```diff
@@ -1218,6 +1218,22 @@ diff(J1, J2) ->
     ?JSON_WRAPPER(L2) = flatten(J2),
     expand(from_list(L1 -- L2)).
 
+-spec flatten_schema(object()) -> object().
+flatten_schema(?JSON_WRAPPER([]) = Empty) -> Empty;
+flatten_schema(?JSON_WRAPPER(L) = Schema) when is_list(L) ->
+    from_list(lists:flatten(flatten_props(get_value(<<"properties">>, Schema), [], Schema))).
```

Comment:
This should be in kz_json_schema - that module understands JSON schema documents. kz_json does not.